content-security-policy/support/report.py

```diff
@@ -13,32 +15,32 @@ def retrieve_from_stash(request, key, timeout, default_value):
   return default_value
 
 def main(request, response):
-  op = request.GET.first("op");
-  key = request.GET.first("reportID")
-  cookie_key = re.sub('^....', 'cccc', key)
-  count_key = re.sub('^....', 'dddd', key)
+  op = request.GET.first(b"op");
+  key = request.GET.first(b"reportID")
+  cookie_key = re.sub(b'^....', b'cccc', key)
+  count_key = re.sub(b'^....', b'dddd', key)
 
   try:
-    timeout = request.GET.first("timeout")
+    timeout = request.GET.first(b"timeout")
   except:
     timeout = 0.5
   timeout = float(timeout)
 
-  if op == "retrieve_report":
-    return [("Content-Type", "application/json")], retrieve_from_stash(request, key, timeout, json.dumps({'error': 'No such report.' , 'guid' : key}))
+  if op == b"retrieve_report":
+    return [(b"Content-Type", b"application/json")], retrieve_from_stash(request, key, timeout, json.dumps({u'error': u'No such report.' , u'guid' : isomorphic_decode(key)}))
 
-  if op == "retrieve_cookies":
-    return [("Content-Type", "application/json")], "{ \"reportCookies\" : " + str(retrieve_from_stash(request, cookie_key, timeout, "\"None\"")) + "}"
+  if op == b"retrieve_cookies":
+    return [(b"Content-Type", b"application/json")], u"{ \"reportCookies\" : " + str(retrieve_from_stash(request, cookie_key, timeout, u"\"None\"")) + u"}"
 
-  if op == "retrieve_count":
-    return [("Content-Type", "application/json")], json.dumps({'report_count': str(retrieve_from_stash(request, count_key, timeout, 0))})
+  if op == b"retrieve_count":
+    return [(b"Content-Type", b"application/json")], json.dumps({u'report_count': str(retrieve_from_stash(request, count_key, timeout, 0))})
 
   # save cookies
-  if hasattr(request, 'cookies') and len(request.cookies.keys()) > 0:
+  if hasattr(request, u'cookies') and len(request.cookies.keys()) > 0:
```

Comment:
You didn't change it, but the `hasattr` bit here looks a bit suspect. `request.cookies` will always be there, it seems to me.

How have you been dealing with minor issues like this in these reviews? Do you fix them when requested, or ask the reviewer to fix it themselves in a follow-up? I'm happy to do the latter.

Reply:
I need to fix them after receiving review comments. 

> You didn't change it
Sorry, what does "it" refer here?

Reply:
I mean there was a `hasattr(request, 'cookies')` part to this condition, which you did change, but what I'm wondering about here is why the overall `hasattr` check is here at all, which you didn't change.

I've sent https://github.com/web-platform-tests/wpt/pull/24196 with the change I had in mind. That will conflict with this change, so I'll wait until this lands before resolving the conflict.

Reply:
I see :). Code updated and all comments should have been addressed.